docs/modules/ROOT/pages/testing.adoc

```diff
@@ -100,23 +100,70 @@ class CreateCommentTest {
     void testIssueOpened() throws IOException {
         GitHubAppTesting.given() // <1>
                 .github(mocks -> { // <2>
-                    Mockito.doThrow(new RuntimeException()) // <3>
+                    Mockito.doThrow(new RuntimeException("Simulated exception")) // <3>
                             .when(mocks.issue(750705278))
                             .comment(Mockito.any());
                 })
                 .when().payloadFromClasspath("/issue-opened.json")
                 .event(GHEvent.ISSUES)
-                .then().github(mocks -> {
-                    Mockito.verify(mocks.issue(750705278))
-                            .comment("Hello from my GitHub App");
+                .then().github(mocks -> { // <4>
+                    Mockito.verify(mocks.issue(750705278)) // <5>
+                            .createReaction(ReactionContent.CONFUSED);
                 });
     }
 }
 ----
 <1> Use given().github(...) to configure mocks.
 <2> The given `mocks` object gives access to mocks of GitHub objects, indexed by their identifier,
     just like in `.then().github(...)`.
+    This can be used to configure the behavior of objects referenced in the event payload,
+    such as (here) the `GHIssue`.
 <3> Here we're configuring the mock to throw an exception when the application tries to comment on the issue.
+<4> We can still use `.then().github(mocks -> ...)` to perform assertions on GitHub objects involved in the event handling
+<5> Here we're verifying that the application caught the runtime exception and added a "confused" reaction to the GitHub issue.
+
+You can also use the class `GitHubAppMockUtils` to simplify mocking for some common scenarios:
```

Comment:
The class name looks off, I will fix it.

Reply:
Thanks